scripts/abi_bindings.sh

```diff
@@ -139,21 +139,30 @@ function generate_bindings() {
 
         # Filter out the contract we are generating bindings for
         filtered_contracts=$(remove_matching_string $contracts $contract_name)
-
-        # Filter out external libraries
-        for lib in $EXTERNAL_LIBS; do
-            filtered_contracts=$(remove_matching_string $filtered_contracts $lib)
-        done
-
-        echo "Generating Go bindings for $contract_name..."
+        
         gen_path=$TELEPORTER_PATH/abi-bindings/go/$dir/$contract_name
         mkdir -p $gen_path
-
-        $GOPATH/bin/abigen --pkg $(convertToLower $contract_name) \
-                        --combined-json $combined_json \
-                        --type $contract_name \
-                        --out $gen_path/$contract_name.go \
-                        --exc $filtered_contracts
+        echo "Generating Go bindings for $contract_name..."
+        
+        if [ -z "$filtered_contracts" ]; then
```

Comment:
This addresses an edge case in which the only contract in the combined JSON is the contract to compile itself. In that case, there are no contracts to exclude.